modules/setting/indexer.go

```diff
@@ -27,20 +27,25 @@ var (
 	Indexer = struct {
 		IssueType             string
 		IssuePath             string
-		RepoIndexerEnabled    bool
-		RepoPath              string
-		UpdateQueueLength     int
-		MaxIndexerFileSize    int64
+		IssueConnStr          string
+		IssueIndexerName      string
 		IssueQueueType        string
 		IssueQueueDir         string
 		IssueQueueConnStr     string
 		IssueQueueBatchNumber int
 		StartupTimeout        time.Duration
-		IncludePatterns       []glob.Glob
-		ExcludePatterns       []glob.Glob
+
```

Comment:
what is this space for?

Reply:
Before this space line it's issue indexer configurations, after this line it's code indexer configurations.